### PR TITLE
update oboe-talk

### PR DESCRIPTION
**Pull request by JuanCaicedo**
@jakerella 

I updated the abstract for my Oboe-talk based on some of the last things you and I talked about. In particular. Let me know any thoughts you have!

### CODE REVIEW

File: oboe-talk.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
-## AJAX++ with Event-Driven JSON loading
+## A better AJAX, loading large JSON easily
```
Comment (jakerella) [inline oboe-talk.md:1]: This is better.... not 100% for me, but definitely an improvement.

File: oboe-talk.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
+Working with large JSON documents in a web app can be slow and cumbersome. But that is no reason to let your user experience suffer! By bringing event-driven JSON parsing to your REST data transfer, using Oboe.js, it is posible to write applications that handle large documents just as easily as small ones. This new way of looking at AJAX makes it possible to handle dropped connections with grace, step outside the trade-off between big and small JSON, and convert any API to a mobile-friendly data source. Doing so enables faster and more reliable web apps, all while making user happier. In this talk, developers will learn how to implement Oboe.js in their client and server-side code and the practical applications for doing so.
```
Comment (jakerella) [inline oboe-talk.md]: It feels like "...using Oboe.js..." is just casually thrown in and dismissed. I'd suggest pulling it out of that third sentence and instead making it its own sentence introducing Oboe as the tool for this task.

**Comment by jakerella** [comment]
Getting better for sure. Be careful to re-read your abstracts for grammar though. ;)

**Comment by JuanCaicedo** [comment]
I'm not sure how to improve on the title. I'm going to merge for now and maybe we can talk about it next time we chat.

**Pull request merged**